addons/explosives/functions/fnc_addExplosiveActions.sqf

```diff
@@ -36,8 +36,7 @@ _itemCount = [];
 _children = [];
 
 {
-    private "_name";
-    _name = if(isText(_x >> "displayNameShort") && {getText(_x >> "displayNameShort") != ""}) then
+    private _name = if(isText(_x >> "displayNameShort") && {getText(_x >> "displayNameShort") != ""}) then
         {getText (_x >> "displayNameShort")}else{getText(_x >> "displayName")};
```

Comment:
I would've written this as:

```sqf
    private _name = getText (_x >> "displayNameShort");

    if (_name isEqualTo "") then {
        _name = getText (_x >> "displayName");
    };

    _children pushBack [
```

displayNameShort is always a string or undefined. the game would complain otherwise.